### ISSUE TITLE
PLEASE NOMINATE MAINTAINERS

### ISSUE THREAD

**Issue by lonix1** [g3offrey/javascript-test-runner#41]
@g3offrey This is an excellent extension - thank you so much for making it! :+1: :+1: :+1:

But there are many bugs and PRs, and the community wants to help you!

If you don't have time to maintain it anymore, consider adding one or two contributors. Maybe choose from those who've made PRs, as they are familiar with the codebase?

What do you think about this idea? Please let us know this extension is not dead! :heart:

**Comment by lonix1** [comment]
If you are willing to help @g3offrey please post below so he knows who is interested...

**Comment by Romstar** [comment]
I am willing to maintain it

**Comment by fabienbc** [comment]
Agreed, many small issues could be fixed easily, and some already have PRs.

And again this extension is a "must have" in my opinion, that's why I would love to see these some of these issues fixed 😇 ❤️ 

Please let me know if I can help @g3offrey

**Comment by g3offrey** [comment]
Hi everyone, 
First of all please accept my apologies for the long time I took to answer. 😢 

These days, I can't contribute as much as I wanted to on this project. 
I also have some others project in my mind I would like to start.

However this project is completely open source with a very permissive license, so don't hesitate to fork this extension and make yours.

I also want to highlight a great extension I found on the marketplace, maybe even better than this one.
https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer
So don't hesitate to try it.

I'll put a message on the readme to announce that the extension is now deprecated, and won't be updated anymore.

Again, please accept my apologies. 

Please be advised there is some security flaws from the dependencies (according dependabot) on the current version. 
I'll remove the extension from the marketplace to protect people.

Big thank you to every people who contributed to this awesome piece of software.
You turned my shitty code into something great. 😅 

Hope to see you soon on another project. 👋

**Comment by g3offrey** [comment]
I reopen this ticket, if you want to synchronize yourself to merge your effort into a new repository.

**Comment by lonix1** [comment]
@g3offrey Instead of retiring this extension, why not sign it over to one of those who want to maintain it? There seems to be quite a few willing to take it over from you. It would be easier than forking and setting up everything from scratch.

**Comment by g3offrey** [comment]
I won't feeling good, providing access to VSCode marketplace which would permit to deploy code on hundreds of machine (through the extension) on my name.
We saw what can happen doing this with npm package hacks that happened these last days. 😅 

There is a build available on this repository freezed at this moment in time.
If you want to continue the work to another direction, a fork would permit to keep both builds.

**Comment by g3offrey** [comment]
I would like to add that there is a lot of extensions about testing on marketplace. 
I don't think it would be a good idea to split our work, when we can contribute to some good ones. 
That was a point that affected me on my decision. 😄 
(That was purely subjective however)

**Comment by lonix1** [comment]
I think a repo can be transferred to a new name, to protect the original owner. I've seen it done before, but I'm unsure how it was done. Maybe someone else wants to look into that.

BTW, you should know that your extension is still the best in its class. The other ones you mentioned are good but different. Thanks for making it, it was fun using it! :smiley:

**Comment by g3offrey** [comment]
The repo can be transferred, however that's the publication on VSCode Marketplace, who still would be on my name. I also prefer to keep a copy of the repo in case I would like to make some changements in the future for my particular use case. 😅 

@lonix1 thank you for these kind words, that means a lot. 😍

**Comment by felixjb** [comment]
Hi @g3offrey, I just wanted to say that I loved to use your extension and I was really sad to see the project deprecated. It's the best extensions I have used so far.

Anyway, I decided to fork and maintain the project. I relaunched the extensions under a new name, Testify. I want to implement the issues and PRs that didn't make it to this project and add new features.

If anyone wants to contribute, feel free to help out: 
[GitHub repo](https://github.com/felixjb/testify)
[VSCode marketplace](https://marketplace.visualstudio.com/items?itemName=felixjb.testify)

P.S.: I'm thinking of using a new icon to the extension because nobody is really getting the movie reference :smile:

**Comment by wisetc** [comment]
Why you change the author of this extension from legfrey to oshri6688 thus cause the vscode-sync extension didn't work. 使用 Code-Settings-Sync 报错了，终止执行。

**Comment by wisetc** [comment]
Why you change the author of this extension from legfrey to oshri6688 thus cause the vscode-sync extension didn't work. 使用 Code-Settings-Sync 报错了，终止执行。
